packages/transformer/src/IModelExporter.ts

```diff
@@ -264,29 +278,48 @@ export class IModelExporter {
     await this.exportRelationships(ElementRefersToElements.classFullName);
   }
 
-  /** Export changes from the source iModel.
-   * @param user The user
-   * @param startChangesetId Include changes from this changeset up through and including the current changeset.
-   * If this parameter is not provided, then just the current changeset will be exported.
-   * @note To form a range of versions to export, set `startChangesetId` for the start (inclusive) of the desired range and open the source iModel as of the end (inclusive) of the desired range.
-   */
-  public async exportChanges(user?: AccessToken, startChangesetId?: string): Promise<void> {
-    if (!this.sourceDb.isBriefcaseDb()) {
+  /**
+   * Export changes from the source iModel.
+   */
+  public async exportChanges(args?: ExportChangesOptions): Promise<void>;
+  /** @deprecated in 0.1.x, use a single ExportChangesArgs object instead */
+  public async exportChanges(accessToken?: AccessToken, startChangesetId?: string, args?: ExportChangesOptions): Promise<void>;
+  /** @internal Don't use this overload. */
```

Comment:
nit: is it necessary to mark this as internal? It's not an overload but the implementation. I think typescript will identify similar usage (more than 1 argument) as the deprecated overload so this is unnecessary

Reply:
Okay, I removed it :)